pennylane/plugins/default_qubit.py

```diff
@@ -328,18 +329,17 @@ def reset(self):
         self._state = self._create_basis_state(0)
         self._pre_rotated_state = self._state
 
-    def analytic_probability(self, subsystems=None):
-        """Return the (marginal) analytic probability of each computational basis state.
```

Comment:
docstring unnecessary?

Reply:
I learnt from @josh146 that functions inherited from a parent class do not need a docstring?

Reply:
Yep!